src/test.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+#include <R.h>
```

Comment:
You probably don't need this include in this file, since you aren't using anything from R.

Reply:
The file shouldn't be in the repository anymore.